File: notesProject/src/Note.kt
package com.alzhanotes

import org.bson.codecs.pojo.annotations.BsonId
import org.litote.kmongo.Id
import org.litote.kmongo.newId
import java.util.*

data class Note (@BsonId val _id: Id<Note> = newId(),
                 val author:String,
                 val body:String,
                 val uniOnly:Boolean,
                 val priority: Int,
                 val date: String) {
}

File: note_app/lib/note.dart
import 'dart:async';
import 'dart:collection';
import 'dart:convert';
import 'package:flutter/material.dart';
import 'package:http/http.dart' as http;

class Note{
  final String author;
  final String body;
  final String date;
  final int priority;
  final bool uniOnly;
  //Note({ this.id, this.author, this.body, this.date, this.priority, this.uniOnly});
  Note( this.author, this.body, this.date, this.priority, this.uniOnly);

}

